code/traditional/TCA/TCA.py

```diff
@@ -65,20 +61,14 @@ def fit(self, Xs, Xt1):
         a, b = np.linalg.multi_dot([K, M, K.T]) + self.lamb * np.eye(n_eye), np.linalg.multi_dot([K, H, K.T])
         w, V = scipy.linalg.eig(a, b)
         ind = np.argsort(w)
-        
-        # Projecting to latent space
         A = V[:, ind[:self.dim]]
         Z = np.dot(A.T, K)
         Z /= np.linalg.norm(Z, axis=0)
         
-        Xs_new, Xt1_new = Z[:, :ns].T, Z[:, ns:].T
-        
-        self.W_proj = A
-        self.X_new_src = X
-        
-        return Xs_new, Xt1_new
+        Xs_new, Xt_new = Z[:, :ns].T, Z[:, ns:].T
+        return Xs_new, Xt_new
 
-    def fit_predict(self, Xs, Ys, Xt1, Yt1):
+    def fit_predict(self, Xs, Ys, Xt, Yt):
         '''
```

Comment:
as per requested I convert all `fit` and `fit_predict` methods and all tis variable name to original, let me know if I miss something